docs/jwst/references_general/abvega_offset_reffile.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
```

Comment:
I think that this scheme of having an (orphan) .rst file include a .inc file is only necessary when the ref file type is used by multiple steps and hence has to be referenced from multiple step docs (as is the case for apcorr, being used by both `source_catalog` and `extract_1d`. But since this ref file type is only used by one step (`source_catalog`) I think you can use the normal doc file layout used by other ref files that correspond to a single step, such as linearity and superbias. In that scheme, you only have the 2 .inc files (abvega_offset_reffile.inc and abvega_offset_selection.inc), where the abvega_offset_reffile.inc file contains the reference ".. _abvega_offset_reffile:" in its first line (i.e. just move line 3 of this file into the file above), and similarly move the line ".. _abvega_offset_selectors:" into abvega_offset_selection.inc, and remove the 2 .rst files.

Reply:
You also need to update the file `docs/jwst/references_general/references_general.rst` to add entires for these new abvega_offset docs into the 2 tables that give reffile vs. step and step vs. reffile. And also update the reference files section of the `source_catalog` step docs to list the new abvega_offset ref file.

Reply:
Thanks, @hbushouse.  I've made those changes.

Reply:
Also, I didn't update the reference files section of the `source_catalog` step in this PR.  I updated that section to include both the `apcorr` and `abvega_offset` reference files in my upcoming `source_catalog` PR that makes use of these reference files.